deepchem/dock/binding_pocket.py

```diff
@@ -24,7 +25,7 @@ def extract_active_site(protein_file, ligand_file, cutoff=4):
     Location of protein PDB
   ligand_file: str
     Location of ligand input file
-  cutoff: int, optional
+  cutoff: float, optional (default 4.0)
```

Comment:
`get_contact_atom_indices` 's annotation is float. So, I changed.  Is it correct...?